core/src/main/java/io/kestra/core/models/flows/Flow.java

```diff
@@ -129,6 +130,10 @@ public List<PluginDefault> getTaskDefaults() {
     @Valid
     @PluginProperty
     List<SLA> sla;
+    
```

Comment:
It would be cool to add your documentation here, so it can be in the json schema, and so later displayed in the UI editor ?

Reply:
Added some docs. 

Actually there is more than `inputs`. But this will be documented by the product team. For the moment, we don't have doc example on the flow classes directly so I'll not start with that PR. That's something that can be done later for all props and part of SDK if it's really necessary.